# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/firefox/i486.patch
# Copyright (C) 2020 - 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-84.0/third_party/rust/target-lexicon-0.9.0/src/targets.rs.vanilla	2020-12-26 16:03:43.098345471 +0100
+++ firefox-84.0/third_party/rust/target-lexicon-0.9.0/src/targets.rs	2020-12-26 16:04:48.562344035 +0100
@@ -16,6 +16,7 @@
     Asmjs,
     Hexagon,
     I386,
+    I486,
     I586,
     I686,
     Mips,
@@ -394,6 +395,7 @@
             | Architecture::Asmjs
             | Architecture::Hexagon
             | Architecture::I386
+            | Architecture::I486
             | Architecture::I586
             | Architecture::I686
             | Architecture::Mips64el
@@ -435,6 +437,7 @@
             Architecture::Asmjs
             | Architecture::Hexagon
             | Architecture::I386
+            | Architecture::I486
             | Architecture::I586
             | Architecture::I686
             | Architecture::Mipsel
@@ -558,6 +561,7 @@
             Architecture::Asmjs => f.write_str("asmjs"),
             Architecture::Hexagon => f.write_str("hexagon"),
             Architecture::I386 => f.write_str("i386"),
+            Architecture::I486 => f.write_str("i486"),
             Architecture::I586 => f.write_str("i586"),
             Architecture::I686 => f.write_str("i686"),
             Architecture::Mips => f.write_str("mips"),
@@ -662,6 +666,7 @@
             "asmjs" => Architecture::Asmjs,
             "hexagon" => Architecture::Hexagon,
             "i386" => Architecture::I386,
+            "i486" => Architecture::I486,
             "i586" => Architecture::I586,
             "i686" => Architecture::I686,
             "mips" => Architecture::Mips,
--- firefox-84.0/third_party/rust/target-lexicon-0.9.0/.cargo-checksum.json.vanilla	2020-12-26 16:14:02.942331872 +0100
+++ firefox-84.0/third_party/rust/target-lexicon-0.9.0/.cargo-checksum.json	2020-12-26 16:14:34.722331175 +0100
@@ -1 +1 @@
-{"files":{"Cargo.lock":"a1a162e6ce8fc2234a6ddf7090410006a1920ace8738772e32a5b50e4780c19d","Cargo.toml":"f3b545fa0f184fd0d3624e6e5c205fcbdf1ad0934a2e08406549ad53c2a62ac3","LICENSE":"268872b9816f90fd8e85db5a28d33f8150ebb8dd016653fb39ef1f94f2686bc5","README.md":"c3467056d91be3f59562158ee9604c729b5b5f473efbefb036032803eb76809e","build.rs":"723100e9cdc30cd8c48407233c2cffa10f5b10703a0a11bac1230d8b86e49ccf","examples/host.rs":"503bafddfb372123fe4dc0e7b8037808beb5bfe6df60c00d3315922bd3792c6c","examples/misc.rs":"49a579845450b7b020ed5c97dca142fc548725893cbc82f6f750ee0caab2beca","newlist":"89564342916321c5bc35e772d374a7f0af22cc9ae6dcc0027eca48d2269f18cb","src/host.rs":"fb543df4f362e9119a58523563e453110f4e3a426f0995911d0ca386657cf1d9","src/lib.rs":"4414353c30f25d44df6cc14f7f9eea9991222289c6aa662b74406f6923235970","src/parse_error.rs":"b3735eabc0fd0a9dfdd6375662f20ec96a79852a00a05a98fb2e421545285e53","src/targets.rs":"9ccc0849cff06d8906dacbdc15136cc47fab85ccd795033ddfdde1397dfcfe32","src/triple.rs":"949bd83b043b53b18f643ebc3fbebbfe02a13998b787fda432a5d36aa27d20bd","test.sh":"22e3c630a6c84e90d5c70c367a6712be8eeca1e7682c00d1f65bf53e330e9191"},"package":"6f4c118a7a38378f305a9e111fcb2f7f838c0be324bfb31a77ea04f7f6e684b4"}
\ No newline at end of file
+{"files":{"Cargo.lock":"a1a162e6ce8fc2234a6ddf7090410006a1920ace8738772e32a5b50e4780c19d","Cargo.toml":"f3b545fa0f184fd0d3624e6e5c205fcbdf1ad0934a2e08406549ad53c2a62ac3","LICENSE":"268872b9816f90fd8e85db5a28d33f8150ebb8dd016653fb39ef1f94f2686bc5","README.md":"c3467056d91be3f59562158ee9604c729b5b5f473efbefb036032803eb76809e","build.rs":"723100e9cdc30cd8c48407233c2cffa10f5b10703a0a11bac1230d8b86e49ccf","examples/host.rs":"503bafddfb372123fe4dc0e7b8037808beb5bfe6df60c00d3315922bd3792c6c","examples/misc.rs":"49a579845450b7b020ed5c97dca142fc548725893cbc82f6f750ee0caab2beca","newlist":"89564342916321c5bc35e772d374a7f0af22cc9ae6dcc0027eca48d2269f18cb","src/host.rs":"fb543df4f362e9119a58523563e453110f4e3a426f0995911d0ca386657cf1d9","src/lib.rs":"4414353c30f25d44df6cc14f7f9eea9991222289c6aa662b74406f6923235970","src/parse_error.rs":"b3735eabc0fd0a9dfdd6375662f20ec96a79852a00a05a98fb2e421545285e53","src/triple.rs":"949bd83b043b53b18f643ebc3fbebbfe02a13998b787fda432a5d36aa27d20bd","test.sh":"22e3c630a6c84e90d5c70c367a6712be8eeca1e7682c00d1f65bf53e330e9191"},"package":"6f4c118a7a38378f305a9e111fcb2f7f838c0be324bfb31a77ea04f7f6e684b4"}
